src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/enable-containerized-function.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Create and deploy New Relic Lambda image layers
+metaDescription: After enabling New Relic's containerized Lambda function, you can create a layer as an image.
+redirects:
+freshnessValidatedDate: never
+---
+
+With containerized Lambda functions, you can create Lambda layers using container images. As an alternative to deploying layers as .zip archives, container images allow for increased flexibility and improved security.
+
+Before you get started, make sure you've built a Dockerfile that includes your chosen runtime environment, installed your dependencies, and copied them into the correct directory structure. 
```

Comment:
I'm extremely confused by the last three paragraphs here. Are they procedure folks should follow first? Things they should have already done? The rest of my feedback could probably be implemented after merging, but I think this should definitely be untangled. Quickfire notes:

- Second paragraph (Before you get started, make sure you..." seems like prereqs. Either make that a header or put it into the procedure itself
- The "intro" drags on and the lack of headers makes the actual procedure below confusing. Either cut a lot of content, add headers, or find some new way to organize it.
- Is the entire first procedure optional? I didn't understand that at first, but it seems that folks can either use the layer in the link OR follow the first procedure. If the entire first procedure is optional, I'd rework it entirely so it doesn't take up the first 50% of the doc (maybe shove it into a clamshell for now until there's more time to work on it?)

Reply:
OK, I think I've addressed all of these points. Super valid and helpful!